### PR TITLE
Edit post

### PR DESCRIPTION
**Pull request by harshitpaliwal95**
## FEAT 
- edit post 
- delete post

### CODE REVIEW

File: src/components/postCard/menu.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import { useState } from "react";
+import Menu from "@mui/material/Menu";
+import MenuItem from "@mui/material/MenuItem";
+import MoreVertIcon from "@mui/icons-material/MoreVert";
```
Comment (Chiragmodi01) [inline src/components/postCard/menu.js]: Can export all 'mui' imports from a separate file and import from a single file.

File: src/components/postCard/postCard.js [hunk 2]
```diff
@@ -108,7 +116,7 @@ export const PostCard = ({ data, authId }) => {
         setLikeCount((pre) => (pre === 1 ? 0 : -1));
       }
     } catch (error) {
-      console.log(error);
+      // console.log(error);
```
Comment (Chiragmodi01) [inline src/components/postCard/postCard.js:125]: maybe add toast at this place? to show user 'the error'

**Review by Chiragmodi01 — COMMENTED** [review]
Code looks good, 
UI seems consistent.
Good to merge!

**Review by agrawalshanu708 — APPROVED** [review]
code looks clean
file structure is good
can merge

File: src/components/postCard/menu.js (hunk 1 above)
Comment (harshitpaliwal95) [reply src/components/postCard/menu.js]: Updated

File: src/components/postCard/postCard.js (hunk 2 above)
Comment (harshitpaliwal95) [reply src/components/postCard/postCard.js:125]: i will add in next pr

**Pull request merged**